siwe/siwe.py

```diff
@@ -210,10 +220,10 @@ def validate(
         try:
             address = w3.eth.account.recover_message(message, signature=signature)
         except eth_utils.exceptions.ValidationError:
-            raise InvalidSignature
+            raise InvalidSignature("Failed to recover public key from signature")
```

Comment:
I'm fairly sure this exception is only raised when the message or signature are invalid (and we control the EIP-191 message generation).